tests/resources/test_dataset.py

```diff
@@ -222,7 +222,7 @@ def test_register_data_concepts(dataset):
         MeasurementSpecCollection: MeasurementSpec("foo"),
         MeasurementRunCollection: MeasurementRun("foo"),
         IngredientSpecCollection: IngredientSpec("foo"),
-        IngredientRunCollection: IngredientRun("foo"),
+        IngredientRunCollection: IngredientRun(),
```

Comment:
I cleaned up some tests as well to get rid of essentially spurious deprecation warnings.